monitoring/docker-compose.yml

```
@@ -0,0 +1,20 @@
+    ports:
+      - "9091:9090"
+    restart: unless-stopped
+
+  grafana:
+    image: grafana/grafana:10.4.2
+    container_name: grafana
+    ports:
+      - "3000:3000"
+    environment:
+      - GF_SECURITY_ADMIN_PASSWORD=admin
+    restart: unless-stopped
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)